airflow/dags/operations/pre_sync_documents_to_kernel_operations.py

```diff
@@ -24,17 +27,37 @@ def get_sps_packages(scilista_file_path, xc_dir_name, proc_dir_name):
     xc_dir_path = Path(xc_dir_name)
     proc_dir_path = Path(proc_dir_name)
     sps_packages_list = []
+    failures = []
 
     with open(scilista_file_path) as scilista:
-        for acron_issue in scilista.readlines():
+        for scilista_item in scilista.readlines():
             # Verifica se comando DEL está indicado no fascículo
-            if not acron_issue.strip().lower().endswith("del"):
-                filename_pattern = "*{}.zip".format("_".join(acron_issue.split()))
-                Logger.info("Reading ZIP files pattern: %s", filename_pattern)
-                for source in sorted(xc_dir_path.glob(filename_pattern)):
-                    Logger.info("Moving %s to %s", str(source), str(proc_dir_path))
-                    shutil.move(str(source), str(proc_dir_path))
+            acron_issue = scilista_item.strip().split()
+            if len(acron_issue) != 2:
+                continue
+            filename_pattern = "*{}.zip".format("_".join(acron_issue))
+            Logger.info("Reading ZIP files pattern: %s", filename_pattern)
+            try:
+                files = list(xc_dir_path.glob(filename_pattern))
+                if len(files) == 0:
+                    raise FileNotFoundError(
+                        "Not found files which pattern is '{}'".format(
+                            filename_pattern))
+                for source in sorted(files):
+                    Logger.info("Copying %s to %s", str(source), str(proc_dir_path))
+                    shutil.copy(str(source), str(proc_dir_path))
                     sps_packages_list.append(str(proc_dir_path / source.name))
+            except FileNotFoundError as e:
+                scilista_item = scilista_item.strip()
+                Logger.exception("Missing SPS Package of %s in '%s': %s",
+                    scilista_item, xc_dir_name, e)
+                failures.append(scilista_item)
```

Comment:
Ao invés desse tratamento de exceção laçada no `if` acima, por que não colocar este tratamento no if e adicionar um else para o `for`? O código ficaria mais simples assim.

Reply:
não tinha pensado nisso